src/backend/oracle-data-api/src/test/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeControllerTest.java

```diff
@@ -372,6 +376,29 @@ private List<Dispute> getAllDisputes(Date olderThan, DisputeStatus excludeStatus
 		return result;
 	}
 
+	/**
+	 * Issues a GET request to /api/v1.0/dispute/updateRequests. The appropriate controller is automatically called by the DispatchServlet
+	 * @throws Exception
+	 */
+	public List<DisputantUpdateRequest> getDisputantUpdateRequests(Long id, DisputantUpdateRequestStatus status) {

```

Comment:
I can help craft a junit test in a subsequent PR. 
This method is just a public method and not called by Java's test suite and so unused at the moment.